Mention the Java liveheap subsampling in troubleshooting (#24446)
DataDog/documentation: content/en/profiler/profiler_troubleshooting/java.md

[Learn how to use override templates.](#creating-and-using-a-jfr-template-override-file)

{{% /tab %}}
{{% tab "Datadog Profiler" %}}

> In case you are using the alpha feature of live heap profiling you can tune the overhead by changing the percentage

Reviewer: ```suggestion
If you are using the alpha feature of live heap profiling, you can tune the overhead by changing the percentage
```

> In case you are using the alpha feature of live heap profiling you can tune the overhead by changing the percentage

Reviewer 2: ```suggestion
If you are using the alpha feature of live heap profiling, you can tune the overhead by changing the percentage
```
When I went to make this suggestion, I noticed this suggestion was added by a previous reviewer but wasn't applied -- this is the correct grammar, so I'm suggesting it again in case you accidentally resolved it last time. Otherwise, if you have objections, happy to chat about the logic behind it!
